interop/client/client.go

```diff
@@ -265,105 +267,108 @@ func main() {
 	}
 	defer conn.Close()
 	tc := testgrpc.NewTestServiceClient(conn)
+	overallDeadline := time.Now().Add(time.Duration(*soakOverallTimeoutSeconds) * time.Second)
+	ctxWithDeadline, cancel := context.WithDeadline(ctx, overallDeadline)
```

Comment:
Sorry, I missed this before: please use the more ergonomic `WithTimeout` instead:

```go
ctxWithDeadline, cancel := context.WithTimeout(ctx, *soakOverallTimeoutSeconds * time.Second)
```